Libs/MRML/Core/vtkMRMLSegmentationNode.cxx

```diff
@@ -460,8 +460,16 @@ void vtkMRMLSegmentationNode::GetRASBounds(double bounds[6])
     {
     this->Segmentation->GetBounds(bounds);
     }
+
 }
 
+//---------------------------------------------------------------------------
+void vtkMRMLSegmentationNode::GetNodeBounds(double bounds[6])
+{
+  Superclass::GetNodeBounds(bounds);
```

Comment:
Sure.

Note that in my opinion it's confusing to have another method named GetRASBounds. It implies that this method does NOT return the bounds in the RAS coordinate which it does (albeit not transformed by any vtkMRMLTransformNode applied to the node). That was why I overloaded the GetRASBounds originally.